src/run.rs

```diff
@@ -15,6 +15,25 @@ fn get_day(matches: &ArgMatches) -> Result<u32, std::num::ParseIntError>
     }
 }
 
+fn get_year(matches: &ArgMatches) -> Result<i32, std::num::ParseIntError>
+{
+    if let Some(year) = matches.get_one::<String>("year")
+    {
+        if year.chars().count() == 2
+        {
+            format!("20{}", year).parse()
+        }
+        else
+        {
+            year.parse()
+        }
```

Comment:
Maybe do some validation on the input as well?
Something like
```rs
year.chars().all(char::is_ascii_digit)
```
to disallow somebody writing -y ff

Reply:
Think it's okay. Couldn't do it exactly like you proposed since `char::is_ascii_digit` is a method on char, but the closure that is passed to all expects the item as a parameter

Reply:
actually I think I am the stupid one this time. `parse` will fail if you write -y ff. So it is not necessary..? 🤔 
However,  should we deal with -y 2012 feks? Maybe have a check that its in the range `2015..<current year>`, OR if its already december` 2015..=<current year>`

What do you think?

Reply:
Yeah that's actually true. I also think parse will fail, so it is redundant.
Yes I agree, I'll add a check for the range. No need to keep on trying anything if we know for sure that there is no advent of code for that year

Reply:
let me know what you think about the new change

Reply:
LGTM